core/controllers/reader_test.py

```diff
@@ -26,6 +27,9 @@
 from core.tests import test_utils
 import feconf
 
+EXPLORATION_TYPE = constants.ACTIVITY_TYPE_EXPLORATION
```

Comment:
Rather than aliasing these I suggest just using constants.ACTIVITY_TYPE_* directly, in order to avoid a proliferation of different names for the same thing (especially since you don't use them that much).